### PR TITLE
[FIX] base,l10n_ch: Display all QR codes in QR bill 

### PR DESCRIPTION
**Pull request by nboulif**
Issue:

  When trying to print a Swiss QR bill, if multiple images are presents
  in document, and they have an URL as src, some pictures will not be
  displayed.
  (Same issue may occur with simple QR code)

Cause:

  It's a known issue with wkhtmltopdf: odoo@2949138
  Also, adding CSS class to body by JS break wkhtmltopdf.

Solution:

  Replace link by base64 image value (use a function to retrieve base64
  image instead of image_url).
  Remove class 'l10n_ch_qr' added by js (no need since CSS file dedicated
  to this report).

  Extra: Alter some CSS for better rendering.

opw-2620082

### CODE REVIEW

**Comment by nboulif** [comment]
ping @oco-odoo

**Comment by nboulif** [comment]
ping @oco-odoo

File: addons/l10n_ch/models/res_bank.py [hunk 1]
```diff
@@ -137,9 +138,22 @@ def build_swiss_code_url(self, amount, currency_name, not_used_anymore_1, debtor
...
 
+    @api.model
+    def build_swiss_code_base64(self, amount, currency_name, not_used_anymore_1, debtor_partner, not_used_anymore_2, structured_communication, free_communication):
+        qr_code_vals = self.build_swiss_code_vals(amount, currency_name, not_used_anymore_1, debtor_partner, not_used_anymore_2, structured_communication, free_communication)
+        try:
+            barcode = self.env['ir.actions.report'].barcode('QR_quiet', '\n'.join(qr_code_vals), width=256, height=256, humanreadable=1)
+        except (ValueError, AttributeError):
+            raise werkzeug.exceptions.HTTPException(description='Cannot convert into barcode.')
+
+        return image_data_uri(base64.b64encode(barcode))
```
Comment (oco-odoo) [inline addons/l10n_ch/models/res_bank.py:156]: Arent't we going to face the same kind of problems with SEPA QR-codes then ? Shouldn't we do the same for them ?
https://github.com/odoo/odoo/blob/12.0/odoo/addons/base/models/res_bank.py#L131

(Note for forward-port: this had nothing to do in base, and got moved to a new account_qr_code_sepa module in v14)

**Review by oco-odoo — CHANGES_REQUESTED** [review]

**Review by oco-odoo — APPROVED** [review]
@robodoo r+

**Comment by nboulif** [comment]
@oco-odoo Thanks!
Little question: is it normal that the fw-port didn't create the PRs?

**Pull request closed without merging**